### PR TITLE
Integrated WIF article page

### PR DESCRIPTION
**Pull request by danielmoll**
Also updated some dependencies, specifically `component-articletemplate` as this will be required to pass the build pipeline tests etc.

### CODE REVIEW

File: example-data/article.json [hunk 1]
```diff
@@ -20,12 +20,13 @@
...
-    "title": "The peril beyond Putin",
-    "flytitle": "If Russia breaks up",
+    "title": "The first 100 days",
+    "flytitle": "If Hillary Clinton is President",
     "byline": "Fiammetta Rocco",
     "byline_location": "Mumbai",
     "bio": "<strong>Fiammetta Rocco</strong>: books and arts editor, <em>The Economist</em>",
-    "slug": "what-if-russia-breaks-up-the-peril-beyond-putin",
+    "slug": "if-hillary-clinton-is-president",
+    "dateline": "MOSCOW",
```
Comment (umbobabo) [inline example-data/article.json]: Moscow?? Maybe this should be changed?

Comment (danielmoll) [inline example-data/article.json]: Dummy data, guess it's always been like that.

**Comment by keithamus** [comment]
This doesn't seem right to me - I think components should be white label, not having specific themes for sites. The site-specific themes should be in their respective repos. My 2¢

**Comment by danielmoll** [comment]
I don't disagree, but that's a larger change than was planned for this sprint, but there isn't a reason this can't be tackled in the next iteration for the full WIF article work.

**Comment by danielmoll** [comment]
FYI, this work is just MVP to get WIF articles visible and themed in WIN.

**Comment by umbobabo** [comment]
You should probably specify what is in scope for the PR and what is not.
Anyway IMO if something is in a folder called "example-data" I expect is part of the demo part of the component.

Did the tests in local success?

**Comment by danielmoll** [comment]
The tests passed locally and also when I ran them through sauce labs.

The example-data/article.json etc is stubbed data that would come back from the api. Happy to rename the folder to stubbed-data as that's perhaps more self explanatory.

**Comment by umbobabo** [comment]
I'm gonna merging this code as is.
I don't see at the moment reusability of this code, the CSS are not following the new design standards we are applying.
I consider "example-data" as an example folder, I will expect that the real data are provided in the context statically or dynamically but anyway is per se a contradiction because the component is really high customised and not portable in other context.

**Pull request merged**